index.html

```diff
@@ -7,12 +7,611 @@
+      </div>
+    </div>
+    <div class="bottom-nav">
+      <div class="middle-bottom-nav">
+        <div title="The Next Web" class="left">
+          <img src="assets/tnw.svg" alt="tnw-photo">
+        </div>
+        <div class="left-middle">
+          <a href="#">LATEST</a>
+          <a href="#">HARD FORK</a>
+          <a href="#">PLUGGED</a>
+          <a href="#">README</a>
+          <a href="#">GROWTH QUARTERS</a>
+          <a href="#">SHIFT</a>
+          <a href="#">NEURAL</a>
+        </div>
+        <div class="right">
+          <div class="search">
+          <a href="#"><img src="https://img.icons8.com/dotty/80/000000/search-more.png" alt="search icon" /></a>
+          </div>
+          <div id="burg" class="burger">
+            <a href="#"> <img src="https://img.icons8.com/android/24/fa314a/menu.png" alt="burger" /></a>
+          </div>
+        </div>
+      </div>
+      <div class="yellow-a">
+        <a href="#">Calling young digital in the Netherlands - Be recognized in the T500 <img
+            src="https://img.icons8.com/fluent-systems-filled/24/000000/long-arrow-right.png" alt="burger icon" />
+
+        </a>
+      </div>
+    </div>
+    <div class="bottom-nav-b">
+
+      <div class="black">
+        <a href="#">News</a>
+        <a href="#">Events</a>
+        <a href="#">Business</a>
+        <a href="#">AMAs</a>
+        <a href="#">Spaces</a>
+        <a href="#">Terms & Conditions</a>
+        <a href="#">About</a>
+        <a href="#">Advertise</a>
+        <a href="#">Jobs</a>
+        <a href="#">Contact</a>
+      </div>
+
+      <div class="white">
+        <a href="#">Latest</a>
+        <a href="#">Hard Fork</a>
+        <a href="#">Plugged</a>
+        <a href="#">readme</a>
+        <a href="#">growth quarters</a>
+        <a href="#">shift</a>
+        <a href="#">neural</a>
+      </div>
+
+      <div class="yellow">
+        <a href="#">Calling young digital in the Netherlands - Be recognized in the T500 </a>
+      </div>
+    </div>
+  </nav>
+</header>
+  <!-- #THE HEAD SECTION-->
+
+  <section class="head">
+
+    <div class="left-head top-right">
+      <h2 class="tnw">TNW</h2>
+      <a href="#"><span class="tnw-img"></span></a> 
+      <p class="date"> Tuesday -- January 26, 2021</p>
+      <a href="#" class="tnw-bottom">Opal cuts off your apps’ internet access to help you stop doomscrolling</a>
+      <a href="#" class="author">by Ivan Mehta <i class="far fa-clock"></i> 1 hour ago 12 <i
+          class="fas fa-share-alt"></i></a>
+    </div>
+
+    <div class="top-right-head left-head">
+      <a href="#" class="tnw-bottom">Google unveils privacy-friendly AI cookie killer</a>
+      <a href="#" class="author">by Tomas David <i class="far fa-clock"></i> 40 minutes ago 8 <i
+          class="fas fa-share-alt"></i></a>
+    </div>
+
+    <div class="bottom-right-head left-head">
+      <a href="#" class="tnw-bottom">This Mac app automatically detects your duplicate files</a>
```

Comment:
Kindly wrap all titles with `<h2>/<h3>` tags then wrap them with the `<a>` tag as in currently